sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -81,7 +81,7 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
 
   // Commands
   assertNotSupportedInStreamingPlan(
-    "commmands",
+    "commands",
     DummyCommand(),
     outputMode = Append,
     expectedMsgs = "commands" :: Nil)
```

Comment:
my change here is probably wrong. would `commands_` be ok?
Often you insert a randomly uppercased letter to indicate you're expecting an error...